src/components/KeywordRadioBtnGroup/KeywordRadioBtnGroup.module.scss

```diff
@@ -1,7 +1,57 @@
-.radioGroupWrapper {
+@use "../../styles/settings" as *;
+
+fieldset.radioGroupWrapper {
   display: flex;
+  position: relative;
+  top: -40px;
+  margin: 0 auto;
+  max-width: 75%;
+  border: none;
 
   .radioWrapper {
     display: flex;
+    margin-right: 1.8rem;
```

Comment:
Any good reason, you are sometimes using `px` and sometimes `rem`?

Reply:
OK I'll ammend the code to use rem only for fonts and px for all the other stuff.